pkg/repositories/models/execution.go

```diff
@@ -60,4 +62,11 @@ type Execution struct {
 	State *int32 `gorm:"index;default:0"`
 	// The resource type of the entity used to launch the execution, one of 'launch_plan' or 'task'
 	LaunchEntity string
+	// Tags associated with the execution
+	Tags []AdminTag `gorm:"many2many:execution_admin_tags;"`
+}
+
+type AdminTag struct {
+	gorm.Model
+	Name string `valid:"length(0|255)"`
```

Comment:
you can drop the `valid` tag as that is not really used anywhere.

Reply:
updated